src/components/Radio/index.tsx

```diff
@@ -56,22 +56,34 @@ const StyledRadio = styled(ReakitRadio)`
   opacity: 0.01;
 `
 
-const Radio = ({
+type Props = {
```

Comment:
Is it possible to have something more expressive because when we will generating type definition Radio will take Props type (which can be confusing)